puzzles/models.py

```diff
@@ -151,6 +156,7 @@ def update_sheets():
     def set_answer(self, answer):
```

Comment:
The answer field is now defunct in the React version because this field assumes single answer per puzzle. We now use the related model field "guesses" (see answers/) for this.

We should actually remove this function once the non-react view is deprecated.

Reply:
`set_status` in `answers/models.py` references `set_answer` in `puzzles/models.py`; if we're deprecating the latter, would we have to change how the puzzle is marked as correct? 

(Is the eventual plan to have a choice for puzzle correctness that users manually change? This might be unnecessary and we can just have an informal policy to only enter correct answers, especially if this is too complicated or clunky UI-wise.)